spec/operators/mergeScan-spec.ts

```diff
@@ -419,4 +419,17 @@ describe('mergeScan', () => {
     expectSubscriptions(inner[1].subscriptions).toBe(ysubs);
     expectSubscriptions(inner[2].subscriptions).toBe(zsubs);
   });
+
+  it('should pass current index to accumulator', () => {
+    const recorded: number[] = [];
+    const expected = [0, 1, 2, 3];
+    const e1 = of('a', 'b', 'c', 'd');
+
+    e1.pipe(mergeScan((acc, x, index) => {
+      recorded.push(index);
+      return of(x);
+    }, 0)).subscribe();
+
+    expect(recorded).to.deep.equal(expected);
```

Comment:
Can you please move expected inside of `equal` here? It will make the test moderately more readable, IMO.